tests/core/test_verify_format_supported.py

```diff
@@ -0,0 +1,21 @@
+from pytest import mark, raises
+
+from sprout.core.verify_format_supported import (
```

Comment:
```suggestion
from sprout.core.verify_is_supported_format import (
```

You'll need to rename the files after though.